### DISCUSSION TITLE
Compiling with different process name

### DISCUSSION THREAD

**Discussion by vahidx4r4x** [XTLS/Xray-core#2183] [General]
Hi,
I would like to compile xray with another process name.
Because Data centers will check the process names.
And if there are process names like xray, x-ui, v2ray, etc  in the processes list, The server will be banned.
I want to compile whole xray kernel with a different process name like for eg. ```xftpclient```
How can I proceed?

Thanks

**Comment by Smallthing** [comment]
。。。。你不会rename吗
